XamarinCommunityToolkit/Behaviors/Validators/MultiValidationBehavior.shared.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Linq;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	[ContentProperty(nameof(Children))]
+	public class MultiValidationBehavior : ValidationBehavior
+	{
+		public static readonly BindableProperty ErrorsProperty =
+			BindableProperty.Create(nameof(Errors), typeof(List<object>), typeof(MultiValidationBehavior), null, BindingMode.OneWayToSource);
+
+		public static readonly BindableProperty ErrorProperty =
+			BindableProperty.CreateAttached(nameof(GetError), typeof(object), typeof(MultiValidationBehavior), null);
+
+		readonly ObservableCollection<ValidationBehavior> children
+			= new ObservableCollection<ValidationBehavior>();
+
+		public MultiValidationBehavior()
+			=> children.CollectionChanged += OnChildrenCollectionChanged;
```

Comment:
Should this also be unhooked somewhere?

Reply:
I thought .NET garbage collector should know how to detect and resolve cycle references.
Was I wrong?

Reply:
Hm yeah since this lives in the shared bit that should be true.